tests/Controller/StatusControllerTest.php

```diff
@@ -2,6 +2,9 @@
 
 namespace Tests\BBC\CliftonBundle\Controller;
 
+use BBC\ProgrammesPagesService\Domain\ValueObject\Pid;
+use Doctrine\DBAL\ConnectionException;
+use Doctrine\DBAL\DBALException;
```

Comment:
Class in similar namespaces can be imported as a group `use Doctrine\DBAL\{ConnectionException, DBALException};`